dashboard/Dockerfile

```diff
@@ -13,13 +13,16 @@ RUN gem install compass
 RUN npm install bower grunt-cli -g &&\
   npm install grunt
 
-RUN cd /opt &&\
-  git clone https://github.com/codedellemc/mars-challenge &&\
-  cd /opt/mars-challenge/dashboard/ &&\
-  npm install &&\
-  bower install --allow-root --force-latest --config.interactive=false &&\
-  grunt build &&\
-  cp -r dist /app
+WORKDIR /opt
+RUN git clone https://github.com/codedellemc/mars-challenge
+
+WORKDIR /opt/mars-challenge/dashboard/
+
+RUN npm install
```

Comment:
Is it good or bad to split these into separate steps? I thought the advice was to combine what you could so you didn't create so many layers...